x/auth/ante/feegrant_test.go

```diff
@@ -280,7 +229,3 @@ func genTxWithFeeGranter(gen client.TxConfig, msgs []sdk.Msg, feeAmt sdk.Coins,
 
 	return tx.GetTx(), nil
 }
-
-func TestAnteTestSuite(t *testing.T) {
-	suite.Run(t, new(AnteTestSuite))
-}
```

Comment:
If you remove this, the test cases in `AnteTestSuite` won't run.

Reply:
This suite is no longer required, since all tests which are in `x/feegrant/ante` moved to `x/auth/ante` and using the test suite from the same package(`x/auth/ante`).